pkg/azuredx/datasource.go

```diff
@@ -97,38 +98,34 @@ func (adx *AzureDataExplorer) CheckHealth(ctx context.Context, req *backend.Chec
 	}, nil
 }
 
-func (adx *AzureDataExplorer) handleQuery(q backend.DataQuery, user *backend.User) backend.DataResponse {
```

Comment:
the `handleQuery` method name is kind of a standard in Grafana for the method that process a query. Can you set it back?